DatadogRUM/Sources/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -523,7 +531,28 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
         let refreshRateInfo = vitalInfoSampler?.refreshRate
         let isSlowRendered = refreshRateInfo?.meanValue.map { $0 < Constants.slowRenderingThresholdFPS }
         let networkSettledTime = networkSettledMetric.value(with: context.applicationStateHistory)
-        let interactionToNextViewTime = interactionToNextViewMetric.value(for: viewUUID)
+        var interactionToNextViewTime = interactionToNextViewMetric.value(for: viewUUID)
+        // Only look for an internal override attribute if native INV calculations are disabled
+        if interactionToNextViewTime == .failure(.disabled),
+           let customInvValue = internalAttributes[RUMInternalAttributes.customINVValue] as? (any BinaryInteger) {
+            let customInvValue = Int64(customInvValue)
```

Comment:
blocking/ We must check for overflow
```suggestion
            let customInvValue = Int64(withNoOverflow: customInvValue)
```